src/app/Topology/styles/base.css

```diff
@@ -289,7 +304,7 @@ Below CSS rules only apply to Topology components
   box-sizing: border-box;
   border-radius: 8px;
   padding: 4px;
-  background-color: #fff;
+  background-color: var(--pf-global--palette--white);
 }
 
 .topology__quicksearch__tab-text {
```

Comment:
Lets try this:

```diff
diff --git a/src/app/Topology/Actions/QuickSearchPanel.tsx b/src/app/Topology/Actions/QuickSearchPanel.tsx
index 90a03828..e8c182b8 100644
--- a/src/app/Topology/Actions/QuickSearchPanel.tsx
+++ b/src/app/Topology/Actions/QuickSearchPanel.tsx
@@ -197,6 +197,7 @@ export const QuickSearchPanel: React.FC<QuickSearchPanelProps> = ({ ...props })
           <Sidebar {...props} tabIndex={0} style={{ height: 'max-content' }} hasGutter>
             <SidebarPanel variant="sticky">
               <Tabs
+                className={"topology__quicksearch__tabs"}
                 isVertical
                 unmountOnExit
                 expandable={{ default: 'nonExpandable', md: 'nonExpandable', lg: 'nonExpandable', sm: 'expandable' }}
diff --git a/src/app/Topology/styles/base.css b/src/app/Topology/styles/base.css
index 3f77a668..cb4972c6 100644
--- a/src/app/Topology/styles/base.css
+++ b/src/app/Topology/styles/base.css
@@ -315,10 +315,22 @@ Below CSS rules only apply to Topology components
   margin: 0 !important;
 }
 
+:where(.pf-theme-dark) .topology__quicksearch__tabs {
+  background-color: var(--pf-global--BackgroundColor--dark-300);
+}
+
+:where(.pf-theme-dark) .topology__quicksearch__tabs .pf-c-label {
+  background-color: var(--pf-global--BackgroundColor--dark-200);
+}
+
 .topology__quicksearch__tab.pf-m-current {
   background-color: var(--pf-global--BackgroundColor--200);
 }
 
+:where(.pf-theme-dark) .topology__quicksearch__tab.pf-m-current {
+  background-color: var(--pf-global--BackgroundColor--100);
+}
+
 :where(.pf-theme-dark) .topology__quicksearch__tab-content {
   background-color: var(--pf-global--BackgroundColor--dark-300);
 }
@@ -330,10 +342,14 @@ Below CSS rules only apply to Topology components
 }

```

![Screenshot from 2023-03-29 02-25-13](https://user-images.githubusercontent.com/68053619/228444674-7c5c68e9-5d69-4ecb-bc77-4f238cdb84a9.png)